magento/product-community-edition/2016-07-19.yaml

```diff
@@ -10,6 +10,6 @@ branches:
         versions: ['>=2.1', '<2.2']
     "2.2":
         time:     2014-02-13 11:12:34
```

Comment:
Looks like the date is wrong here, it must be the date of the commit that fixes the issue.